packages/preview/tada/0.1.0/typst.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "tada"
+version = "0.1.0"
+entrypoint = "lib.typ"
+authors = ["Nathan Jessurun"]
+license = "Unlicense"
+description = "Easy, composable tabular data manipulation"
+repository = "https://github.com/ntjess/typst-tada"
+keywords = [
+    "record",
+    "dataframe",
+    "table",
+    "tabular",
+    "agg",
+    "aggregate",
+    "filter",
+    "pandas",
+    "polars",
+]
+
```

Comment:
Now that the `manual.pdf` is part of the submission here, it should be excluded (`exclude = ["manual.pdf"]`) from the package itself.